src/python_framework_v02/troute/nhd_io.py

```diff
@@ -648,14 +648,18 @@ def get_usgs_from_time_slices_csv(routelink_subset_file, usgs_csv):
 
 def get_usgs_from_time_slices_folder(
     routelink_subset_file,
+    run_parameters,
     usgs_files,
-    max_fill_1min = 14
+    data_assimilation_parameters
```

Comment:
In general, we want to call these functions in nhd_io with just bare parameters. The dict.get should be done on the way in and then the function just uses the default or the value passed in.